examples/02_decoding/plot_miyawaki_encoding.py

```diff
@@ -0,0 +1,233 @@
+# Mapping the encoding scores on the brain
+# ----------------------------------------
+# To plot the scores onto the brain, we use :func:`nilearn.image.new_img_like`
+# to create a Nifti1Image containing the scores and then threshold it:
+
+from nilearn.image import threshold_img, new_img_like
+from nilearn.image.resampling import coord_transform
+
+cut_score = np.array(scores).mean(axis=0)
+cut_score[cut_score < 0] = 0
+
+# bring the scores into the shape of the background brain
+score_map = new_img_like(dataset.background,
+                         masking.unmask(cut_score, dataset.mask).get_data())
+
+thresholded_score_map = threshold_img(score_map, threshold = 1e-6)
+
+##############################################################################
+# Plotting the statistical map on a background brain, we mark four voxels
+# which we will inspect more closely later on.
+
+def index_to_xy_coord(x,y,z=10):
+    '''Transforms data index to coordinates of the background + offset'''
+    coords = coord_transform(x,y,z,affine=thresholded_score_map.get_affine())
+    return coords + np.array([0,1,0])
+
+
+xy_indices_of_special_voxels = [(30, 10), (32, 10), (31, 9), (31, 10)]
+
+statmap = plot_stat_map(thresholded_score_map, bg_img=dataset.background,
+                        cut_coords=[-8], display_mode='z', aspect=1.25,
+                        title='Explained variance per voxel',
+                        symmetric_cbar=False)
+
+# creating a contour for each voxel and adding it to the statistical map
+
+for i, (x, y) in enumerate(xy_indices_of_special_voxels):
+    statmap.add_markers(np.array(index_to_xy_coord(x,y))[np.newaxis,:],
+                        edgecolor=['r', 'b', 'magenta','g'][i],
+                        marker_size=150, marker = 's',
+                        facecolor = 'none', lw = 4.5)
+
+
+# re-set figure size after construction so colorbar gets rescaled too
+fig = plt.gcf()
+fig.set_size_inches(12,12)
+
+plt.show()
+
+##############################################################################
+# Estimating receptive fields
+# ---------------------------
+# Now we take a closer look at the receptive fields of the four marked voxels.
+# A voxel's `receptive field <http://en.wikipedia.org/wiki/Receptive_field>`_
+# is the region of a stimulus (like an image) where the presence of an object,
+# like a white instead of a black pixel, results in a change in activity
+# in the voxel. In our case the receptive field is just the vector of 100
+# regression  coefficients (one for each pixel) reshaped into the 10x10
+# form of the original images. Some voxels are receptive to only very few
+# pixels, so we use Lasso regression to estimate a sparse set of
+# regression coefficients.
+
+from sklearn.linear_model import LassoLarsCV
+
+# automatically estimate the sparsity by cross-validation
+lasso = LassoLarsCV(max_iter=10)
+
+# Mark the same pixel in each receptive field
+p = (4,2)
```

Comment:
PEP8: space after coma

Also, any chance that you find a more explicit name for this variable?